.github/workflows/build_osgeo4w.sh

```diff
@@ -25,27 +25,29 @@ fi
 
 test -d "$1" && cd "$1"
 
-osgeo4w_path=/c/OSGeo4W64
+osgeo4w_path=/c/OSGeo4W
 arch=x86_64-w64-mingw32
 src=$(pwd)
 
 # compile
 
-export PATH=/mingw64/bin:/c/OSGeo4W64/bin:/usr/bin
-export PROJ_LIB=$osgeo4w_path/share/proj
+export PATH=/mingw64/bin:/c/OSGeo4W/bin:/usr/bin
```

Comment:
```suggestion
export PATH=/c/OSGeo4W/bin:/mingw64/bin:/usr/bin
```

Make sure Python from OSGeo4W comes first....